inception-imls-dl4j/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/dl4j/pos/DL4JSequenceRecommender.java

```diff
@@ -527,7 +535,9 @@ public double evaluate(List<CAS> aCas, DataSplitter aDataSplitter)
                 }
             }
             
-            return correct / total;
+            double accuracy = correct / total;
```

Comment:
I'd inline this: `result.setDefaultScore(correct / total);` and add a comment that accuracy is calculated here.